pkg/apis/options/common.go

```diff
@@ -0,0 +1,14 @@
+package options
+
+// SecretSource references an individual secret value.
+// Only one source within the struct should be defined at any time.
+type SecretSource struct {
+	// Value expects a base64 encoded string value.
+	Value []byte
```

Comment:
Should we name this Base64Value to make it clearer in other invocations throughout the code that this is Base64 encoded and not the final value to use?